packages/@okta/vuepress-site/docs/api/resources/groups/index.md

```diff
@@ -849,10 +849,7 @@ curl -v -X PUT \
 
 <ApiOperation method="delete" url="/api/v1/groups/${groupId}" />
 
-Removes a group with `OKTA_GROUP` type from your organization.
-
-> Only groups with `OKTA_GROUP` type can be removed.<br>
-> Application imports are responsible for removing groups with `APP_GROUP` type such as Active Directory groups.
+Removes a group with `OKTA_GROUP` or `APP_GROUP` type from your organization.
```

Comment:
Throughout the rest of the document we explicitly say "we don't allow <operation> on app groups", so it might be worth being explicit here that app group deletion is allowed, but I'm not sure what the norm is. @brianzuzga-okta Are there any group types other than APP_GROUP and OKTA_GROUP? Maybe LDAP groups?

Reply:
`OKTA_GROUP` or `APP_GROUP` should cover everything.